projects/addon-doc/components/demo/demo.style.less

```diff
@@ -5,37 +5,44 @@
 @color-dark-resizer: #3e464e;
 
 :host {
+    .shadow(2);
+
     position: relative;
     display: block;
     min-width: 100%;
-    margin: 1.5rem 0;
+    margin: 0 0 1.5rem;
+    border: 1px solid var(--tui-base-03);
+    border-radius: var(--tui-radius-m);
+    background: var(--tui-base-02);
+}
+
+.t-tabs {
+    padding: 0 1.5rem;
+    background: var(--tui-base-01);
+    border-radius: var(--tui-radius-m) var(--tui-radius-m) 0 0;
 }
 
-.t-bg-toggle {
-    position: absolute;
-    bottom: ~'calc(100% + 0.75rem)';
+.t-options {
+    min-width: 10rem;
+}
+
+.t-mode-option {
     display: flex;
     align-items: center;
-    justify-content: flex-end;
-    pointer-events: none;
-    width: 100%;
-
-    & > * {
-        pointer-events: auto;
-    }
+    gap: 0.5rem;
 }
 
 .t-wrapper {
-    .shadow(2);
     display: flex;
-    border: 1px solid var(--tui-base-03);
-    border-radius: var(--tui-radius-m);
     max-width: 100%;
     min-height: 6rem;
     width: 100%;
     min-width: 10rem;
     background-color: var(--tui-base-01);
     overflow: hidden;
+    position: relative;
+    z-index: 2;
```

Comment:
Are `position` and `z-index` relative? Doesn't seem like they do anything.

Reply:
This is not need anymore after redesign. Fixed.